source-monday/source_monday/models.py

```diff
@@ -0,0 +1,363 @@
+            return datetime.fromtimestamp(unix_millis / 1_000, tz=UTC)
+        raise ValueError(f"Invalid created_at value: {v}")
+
+
+class BoardActivityLogs(BaseModel, extra="forbid"):
+    id: str
+    activity_logs: list[ActivityLog]
+
+
+class ActivityLogsResponse(BaseModel, extra="forbid"):
+    boards: list[BoardActivityLogs]
+
+
+# A simple reference model for entities with just an id.
+class EntityRef(BaseModel, extra="forbid"):
+    id: str
+
+
+class Column(BaseModel, extra="forbid"):
+    archived: bool
+    description: str | None = None
+    id: str
+    settings_str: str | None = None
+    title: str
+    type: str
+    width: int | None = None
+
+
+class Group(BaseModel, extra="forbid"):
+    archived: bool
+    color: str | None = None
+    deleted: bool
+    id: str
+    position: int | None = None
+    title: str
+
+
+class Tag(BaseDocument, extra="forbid"):
+    id: str
+    color: str
+    name: str
+
+
+class View(BaseModel, extra="forbid"):
+    id: str
+    name: str
+    settings_str: str | None = None
+    type: str
+    view_specific_data_str: str | None = None
+
+
+class Workspace(BaseModel, extra="forbid"):
+    id: str
+    name: str
+    kind: str
+    description: str | None = None
+
+
+class Board(BaseDocument, extra="forbid"):
+    id: str
+    name: str
+    board_kind: str
+    type: str
+    columns: list[Column]
+    communication: str | None = None
+    description: str | None = None
+    groups: list[Group]
+    owners: list[EntityRef]
+    creator: EntityRef
+    permissions: str | None = None
+    state: str
+    subscribers: list[EntityRef]
+    tags: list[Tag]
+    top_group: EntityRef
+    updated_at: AwareDatetime
+    updates: list[EntityRef]
+    views: list[View]
+    workspace: Workspace
+
+
+class BoardsResponse(BaseModel, extra="forbid"):
+    boards: list[Board]
+
+
+class BoardRef(BaseModel, extra="forbid"):
+    id: str
+    name: str
+
+
+class Asset(BaseModel, extra="forbid"):
+    created_at: AwareDatetime
+    file_extension: str
+    file_size: int
+    id: str
+    name: str
+    original_geometry: str | None = None
+    public_url: str
+    uploaded_by: EntityRef
+    url: str
+    url_thumbnail: str
+
+
+class ColumnValue(BaseModel, extra="forbid"):
+    id: str
+    text: str | None = None
+    type: str
+    value: str | None = None
+
+
+class GroupRef(BaseModel, extra="forbid"):
+    id: str
+
+
+class ParentItemRef(BaseModel, extra="forbid"):
+    id: str
+
+
+class Update(BaseModel, extra="forbid"):
+    id: str
+
+
+class Item(BaseDocument, extra="forbid"):
+    id: str
+    name: str
+    assets: list[Asset]
+    board: BoardRef
+    column_values: list[ColumnValue]
+    created_at: AwareDatetime
+    creator_id: str
+    group: GroupRef
+    parent_item: ParentItemRef | None = None
+    state: str
+    subscribers: list[EntityRef]
+    updated_at: AwareDatetime
+    updates: list[Update]
+    subitems: list["Item"] | None = None
+
+
+Item.model_rebuild()
```

Comment:
Not sure what this does?

Reply:
Hmm we have lots of examples of using forward references that just put the class name in quotes. What happens if you get rid of this `Item.model_rebuild` line?

Reply:
Hmm you are right. I guess I did not need that after all. I will update to remove that line since I just tested without and everything worked as expected!